pkg/liqoctl/info/peer/auth.go

```diff
@@ -0,0 +1,217 @@
+			}
+
+			// Show resource slices
+			slicesSection := main.AddSection("Resource slices")
+			for i := range data.ResourceSlices {
+				slice := &data.ResourceSlices[i]
+				currSliceSection := slicesSection.AddSection(slice.Name)
+				if slice.Accepted {
+					currSliceSection.AddSectionSuccess("Resource slice accepted")
+				} else {
+					currSliceSection.AddSectionFailure("Resource slice not accepted")
+				}
+				if len(slice.Alerts) > 0 {
+					main.AddEntryWarning("Alerts", slice.Alerts...)
+				}
+				currSliceSection.AddEntry("Action", string(slice.Action))
+
+				resourcesSection := currSliceSection.AddSection("Resources")
+				for resource, quantity := range slice.Resources {
+					resourcesSection.AddEntry(string(resource), quantity.String())
+				}
+			}
+		}
+
+		return main.SprintForBox(options.Printer)
+	}
+	return ""
+}
+
+// GetData returns the data collected by the checker.
+func (ac *AuthChecker) GetData() interface{} {
+	return ac.data
+}
+
+// GetDataByClusterID returns the data collected by the checker for the cluster with the give ClusterID.
+func (ac *AuthChecker) GetDataByClusterID(clusterID liqov1beta1.ClusterID) (interface{}, error) {
+	if res, ok := ac.data[clusterID]; ok {
+		return res, nil
+	}
+	return nil, fmt.Errorf("no data collected for cluster %q", clusterID)
+}
+
+// GetID returns the id of the section collected by the checker.
+func (ac *AuthChecker) GetID() string {
+	return "authentication"
+}
+
+// GetTitle returns the title of the section collected by the checker.
+func (ac *AuthChecker) GetTitle() string {
+	return "Authentication"
+}
+
+// collectStatusInfo collects the info about the status of the network between the local cluster and the one with the given ClusterID.
+func (ac *AuthChecker) collectStatusInfo(clusterID liqov1beta1.ClusterID,
+	clusterInfo map[liqov1beta1.ClusterID]*liqov1beta1.ForeignCluster, authStatus *Auth) {
+	cluster := clusterInfo[clusterID]
+	authStatus.Status, authStatus.Alerts = common.CheckModuleStatusAndAlerts(cluster.Status.Modules.Authentication)
+}
+
+// collectResourceSlices collect the data from the list of ResourceSlice CRs.
+func (ac *AuthChecker) collectResourceSlices(resourceSlices []authv1beta1.ResourceSlice, authStatus *Auth) {
+	authStatus.ResourceSlices = []ResourceSliceStatus{}
+	for i := range resourceSlices {
+		resSlice := &resourceSlices[i]
+
+		// Check the status of the ResourceSlice
+		accepted := false
+		alerts := []string{}
+		for _, condition := range resSlice.Status.Conditions {
+			if condition.Status == authv1beta1.ResourceSliceConditionAccepted {
+				accepted = true
+			} else {
+				alerts = append(alerts, condition.Message)
+			}
+		}
+
+		// To check who is the provider of the ResourceSlice we can verify whether the CR has been replicated from the cluster consumer
+		action := ConsumingAction
+		if _, isProvider := resSlice.ObjectMeta.Labels[consts.ReplicationStatusLabel]; isProvider {
+			action = ProvidingAction
+		}
```

Comment:
Can you also check if the value is set to "false" (case insensitive)?
At the moment it does not impact as we never set it to false, but I would change it to be more resilient in case of future changes.